File: NativeApp/Services/APIConstants.swift
import Foundation

struct APIConstants {
    struct DeepSeek {
        static let balanceURL = "https://api.deepseek.com/user/balance"
        static let chatCompletionsURL = "https://api.deepseek.com/chat/completions"
        static let dashboardURL = "https://platform.deepseek.com/usage"
        static let defaultModel = "deepseek-chat"
        static let lowBalanceThreshold: Double = 1.00
    }
    
    struct Ollama {
        static let defaultHost = "http://localhost:11434"
        static let usageURL = "https://ollama.com/api/usage"
        static let settingsURL = "https://ollama.com/settings"
        static let lowSessionQuotaThresholdPercent: Int = 15
    }
    
    struct OllamaPay {
        static let usageTotalURL = "https://ollama-pay.thaigqsoft.com/api/v1/usage/total"
        static let dashboardURL = "https://ollama-pay.thaigqsoft.com/"
    }
}

File: WindowsApp/AIWidgetWindowsApp/Services/APIConstants.cs
namespace AIWidgetWindowsApp.Services
{
    public static class APIConstants
    {
        public static class DeepSeek
        {
            public const string BalanceURL = "https://api.deepseek.com/user/balance";
            public const string ChatCompletionsURL = "https://api.deepseek.com/chat/completions";
            public const string DashboardURL = "https://platform.deepseek.com/usage";
            public const string DefaultModel = "deepseek-chat";
            public const double LowBalanceThreshold = 1.00;
        }

        public static class Ollama
        {
            public const string UsageURL = "https://ollama.com/api/usage";
            public const string SettingsURL = "https://ollama.com/settings";
            public const int LowSessionQuotaThresholdPercent = 15;
        }

        public static class OllamaPay
        {
            public const string UsageTotalURL = "https://ollama-pay.thaigqsoft.com/api/v1/usage/total";
            public const string DashboardURL = "https://ollama-pay.thaigqsoft.com/";
        }
    }
}
